reference/employer/openapi.yaml

```diff
@@ -372,7 +372,10 @@ paths:
                               nullable: true
                             ssn:
                               type: string
-                              description: 'Social Security number of the individual. This field is only available with the `ssn` scope enabled and the `options: { include: [''ssn''] }` param set in the body.'
+                              description: 'Social Security Number of the individual. This field is only available with the `ssn` scope enabled and the `options: { include: [''ssn''] }` param set in the body.'
+                            encrypted_ssn:
```

Comment:
should this be nullable?

Reply:
Yes, I wonder why ssn is not set to `nullable: true`